tests/test_cmdline/__init__.py

```diff
@@ -11,10 +11,11 @@ def setUp(self):
         self.env['SCRAPY_SETTINGS_MODULE'] = 'tests.test_cmdline.settings'
 
     def _execute(self, *new_args, **kwargs):
+        encoding = (os.environ.get('PYTHONIOENCODING') or 'utf-8').rsplit(':')[0]
```

Comment:
probably even getattr(sys.stdout, "encoding") or "utf8" because sys.stdout can be replaced by some packages to an object without 'encoding' attribute